.github/workflows/run-network-tests.yml

```diff
@@ -65,7 +66,7 @@ jobs:
       - name: Cache docker images
         uses: actions/cache@v3
         env:
-          cache-name: joystream-node-docker
+          cache-name: joystream-node-docker-1
```

Comment:
why there's a need for the -1 prefix?

Reply:
Here I just wanted to force github to use a new cache and not rely on previously cached artifacts